zokrates_js/README.md

```diff
@@ -20,7 +20,7 @@ npm install zokrates-js
 ## Usage
 
 ```js
-import { initialize } from 'zokrates-js';
+import { initialize } from 'zokrates-js'; // for node: const { initialize } = require('zokrates-js/node');
```

Comment:
Maybe it would be clearer to have two separate code blocks to describe how to import it in browser and nodejs, then this common code block to show usage?

Reply:
right, makes sense!